pkg/controller/bookkeepercluster/bookkeepercluster_controller.go

```diff
@@ -197,6 +199,20 @@ func (r *ReconcileBookkeeperCluster) deployBookie(p *bookkeeperv1alpha1.Bookkeep
 		return err
 	}
 
+	if !fileExists(util.PravegaMountPath) {
```

Comment:
This solution does not look right. When we have the same bookkeeper-operator managing multiple Bookkeeper Clusters this file will be overwritten with the value for the BKCluster last reconciled by the operator.